gravitee-am-jwt/src/main/java/io/gravitee/am/jwt/DefaultJWTBuilder.java

```diff
@@ -98,7 +109,7 @@ public String sign(JWT payload) {
             if (issuer != null && !payload.containsKey(Claims.iss)) {
                 payload.setIss(issuer);
             }
-            SignedJWT signedJWT = new SignedJWT(header, JWTClaimsSet.parse(payload));
+            SignedJWT signedJWT = new SignedJWT(header, JWTClaimsSet.parse(GSON.toJson(payload)));
```

Comment:
is it not possible to use ObjectMapper?

Reply:
There is no Jackson library imported to use ObjectMapper